Scripts/Macros.cmd.txt

```diff
@@ -0,0 +1,7 @@
+b=%VFS_SCRIPTSDIR%\BuildGVFSForWindows.bat $*
+br=%VFS_SCRIPTSDIR%\BuildGVFSForWindows.bat Release $*
+ut=%VFS_SCRIPTSDIR%\RunUnitTests.bat $*
+utr=%VFS_SCRIPTSDIR%\RunUnitTests.bat Release $*
+ft=%VFS_SCRIPTSDIR%\RunFunctionalTests.bat $*
+ftr=%VFS_SCRIPTSDIR%\RunFunctionalTests.bat Release $*
+scorch=%VFS_SCRIPTSDIR%\NukeBuildOutputs.bat $*

```

Comment:
This script is really intended for build machines, not dev machines. It's not _too_ far off, except that I don't want it killing my GVFS.Service Windows service just because I'm deleting my build outputs.